contrib/for_review/ModifiedBasicPreset/test/IECore/BasicPreset.py

```diff
@@ -196,7 +196,7 @@ def testClassLoader( self ) :
 		messageHandler = IECore.CapturingMessageHandler()
 		with messageHandler :
 
-			loader = IECore.ClassLoader( IECore.SearchPath( savePath, ":" ) )
+			loader = IECore.ClassLoader( IECore.SearchPath( savePath, os.pathsep ) )
```

Comment:
I think we can just omit the second argument here instead, and let the default constructor just choose the right separator. The two-argument version is deprecated because it just gives people the opportunity to get it wrong.

Reply:
Although I realise this file is in `contrib_/for_review`, and really doesn't need maintaining (we could just delete it at this point I think).

Reply:
Sounds good, I removed the entire directory for `ModifiedBasicPreset` in 834c6b060cf51cc4481a757949c34d586bab8e54 assuming if the test goes, so should the target of the test, but I can add back in if needed.